rocksdb_admin/admin_handler.cpp

```diff
@@ -929,6 +982,184 @@ void AdminHandler::async_tm_addS3SstFilesToDB(
   callback->result(AddS3SstFilesToDBResponse());
 }
 
+void AdminHandler::async_tm_startMessageIngestion(
+    std::unique_ptr<apache::thrift::HandlerCallback<std::unique_ptr<
+      StartMessageIngestionResponse>>> callback,
+    std::unique_ptr<StartMessageIngestionRequest> request) {
+
+  admin::AdminException e;
+  e.errorCode = AdminErrorCode::DB_ADMIN_ERROR;
+
+  const auto db_name = request->db_name;
+  const auto topic_name = request->topic_name;
+  const auto kafka_broker_serverset_path = request->kafka_broker_serverset_path;
+  const auto replay_timestamp_ms = request->replay_timestamp_ms;
+
+  LOG(INFO) << "Called startMessageIngestion for db: " << db_name << ", "
+            << "topic_name: " << topic_name << ", "
+            << "serverset path: " << kafka_broker_serverset_path << ", "
+            << "replay_timestamp_ms: " << replay_timestamp_ms;
+
```

Comment:
we may want to call getDB() here. So that we can fail the request early if db doesn't exist.